addons/csw/functions/fnc_reload_loadMagazine.sqf

```diff
@@ -19,46 +20,32 @@
  * Public: No
  */
 
-params ["_vehicle", "_turret", "_carryMag", "_magSource", "_unit"];
-TRACE_5("loadMagazine",_vehicle,_turret,_carryMag,_magSource,_unit);
+params ["_vehicle", "_turret", "_carryMag", "_magSource", "_unit", "_ammo"];
+TRACE_6("loadMagazine",_vehicle,_turret,_carryMag,_magSource,_unit,_ammo);
 
-private _timeToLoad = 1;
-if (!isNull(configOf _vehicle >> QUOTE(ADDON) >> "ammoLoadTime")) then {
-    _timeToLoad = getNumber(configOf _vehicle >> QUOTE(ADDON) >> "ammoLoadTime");
-};
+private _timeToLoad = GET_NUMBER(configOf _vehicle >> QUOTE(ADDON) >> "ammoLoadTime", 1);
 
-private _displayName = format [localize LSTRING(loadX), getText (configFile >> "CfgMagazines" >> _carryMag >> "displayName")];
+private _displayName = format [LLSTRING(loadX), getText (configFile >> "CfgMagazines" >> _carryMag >> "displayName")];
 
 private _onFinish = {
-    (_this select 0) params ["_vehicle", "_turret", "_carryMag", "_magSource", "_unit"];
-    TRACE_5("load progressBar finish",_vehicle,_turret,_carryMag,_magSource,_unit);
+    (_this select 0) params ["_vehicle", "_turret", "_carryMag", "_magSource", "_unit", "_ammo"];
+    TRACE_6("load progressBar finish",_vehicle,_turret,_carryMag,_magSource,_unit,_ammo);
 
-    ([_vehicle, _turret, _carryMag, _magSource] call FUNC(reload_canLoadMagazine)) params ["", "", "_neededAmmo", ""];
```

Comment:
Why was `_magSource` removed?

Reply:
Answer as soon as I remember what the hell I was doing